ParisBNF/abb/BNFabb202.xml

```diff
@@ -0,0 +1,483 @@
+                                  (more recent than <ref target="#p1"/>) but before 1848. Originally
+                                  possibly without binding.</origDate>
+                            </origin>
+                        </history>
+                    </msPart>
+                  </msDesc>
+
+                                <listBibl type="catalogue">
+                                   <bibl>
+                                      <ptr target="bm:ContiRossini1914Notice"/>
+                                      <citedRange unit="page">42-44</citedRange>
+                                      <citedRange unit="number">16</citedRange>
+                                      </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Chaine1912Catalogue"/>
+                                      <citedRange unit="page">120</citedRange>
+                                      <citedRange unit="number">202</citedRange>
+                                   </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Abbadie1859Cat"/>
+                                      <citedRange unit="page">203</citedRange>
+                                      <citedRange unit="number">202</citedRange>
+                                   </bibl>
+                                     </listBibl>
+
+
+                  <listPerson>
+                     <person>
+                        <persName ref="PRS1071dAbbadi" role="owner"/>
+                     </person>
+                     <person>
+                        <persName ref="PRS1071dAbbadi" role="bequeather"/>
+                     </person>
+                   </listPerson>
+               </sourceDesc>
+            </fileDesc>
+            <encodingDesc>
+               <projectDesc>
+                  <p>Encoded according to TEI P5 Guidelines.</p>
+               </projectDesc>
+               <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                  <xi:fallback>
+                     <p>Definitions of prefixes used.</p>
+                  </xi:fallback>
+               </xi:include>
+            </encodingDesc>
+            <profileDesc>
+               <textClass>
+                  <keywords>
+                     <term key="ChristianLiterature"/>
+                     <term key="OldTestament"/>
+                     <term key="Apocrypha"/>
+                     <term key="Prayers"/>
+
+                  </keywords>
+               </textClass>
+               <langUsage><language ident="en">English</language><language ident="ar">Arabic</language><language ident="gez">Gǝʿǝz</language><language ident="am">Amharic</language></langUsage>
+            </profileDesc>
+            <revisionDesc>
+               <change who="DR" when="2018-02-26">Recorded manuscript</change>
```

Comment:
I think the changes are copy pasted from another record.

Reply:
Sorry. Thank you for corrections! :)